### PR TITLE
docs(cn): translate reference/react/createRef into Chinese 

### PR DESCRIPTION
**Pull request by jrr997**
翻译createRef

### CODE REVIEW

File: src/content/reference/react/createRef.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ class Form extends Component {
 }
 ```
 
-If you now pass `ref={this.inputRef}` to an `<input>` in your JSX, React will populate `this.inputRef.current` with the input DOM node. For example, here is how you make a button that focuses the input:
+如果你现在将 `ref={this.inputRef}` 传递给 JSX 中的 `<input>`，React 将把 input 的 DOM 节点填充到 `this.inputRef.current`。例如，以下是如何创建一个将焦点集中到输入框中的按钮的示例代码：
```
Comment (Yucohny) [inline src/content/reference/react/createRef.md:79]:
```suggestion
如果你现在将 `ref={this.inputRef}` 传递给 JSX 中的 `<input>`，React 将把 input 的 DOM 节点填充到 `this.inputRef.current`。例如，以下是创建一个将焦点集中到输入框中的按钮的示例代码：
```

**Review by Yucohny — DISMISSED** [review]

File: src/content/reference/react/createRef.md [hunk 2]
```diff
@@ -40,31 +40,31 @@ class MyComponent extends Component {
...
+`createRef` 不接受任何参数。
 
-#### Returns {/*returns*/}
+#### 返回值 {/*returns*/}
 
-`createRef` returns an object with a single property:
+`createRef` 返回一个对象，该对象只有一个属性：
 
-* `current`: Initially, it's set to the `null`. You can later set it to something else. If you pass the ref object to React as a `ref` attribute to a JSX node, React will set its `current` property.
+* `current`：最初，它被设置为 `null`。你可以稍后将其设置为其他内容。如果你把 ref 对象作为 JSX 节点的 `ref` 属性传递给 React，React 将设置其 `current` 属性。
```
Comment (loveloki) [inline src/content/reference/react/createRef.md:53]:
```suggestion
* `current`：初始值为 `null`，你可以稍后设置为其他内容。如果你把 ref 对象作为 JSX 节点的 `ref` 属性传递给 React，React 将设置其 `current` 属性。
```

File: src/content/reference/react/createRef.md [hunk 3]
```diff
@@ -76,7 +76,7 @@ class Form extends Component {
 }
 ```
 
-If you now pass `ref={this.inputRef}` to an `<input>` in your JSX, React will populate `this.inputRef.current` with the input DOM node. For example, here is how you make a button that focuses the input:
+如果你现在将 `ref={this.inputRef}` 传递给 JSX 中的 `<input>`，React 将把 input 的 DOM 节点填充到 `this.inputRef.current`。例如，以下是创建一个将焦点集中到输入框中的按钮的示例代码：
```
Comment (loveloki) [inline src/content/reference/react/createRef.md:79]:
```suggestion
如果你现在将 `ref={this.inputRef}` 传递给 JSX 中的 `<input>`，React 将把 input 的 DOM 节点赋值给 `this.inputRef.current`。例如，以下是创建一个将焦点集中到输入框中的按钮的示例代码：
```

后面那句没有想到更好的表达

**Comment by jrr997** [comment]
1

**Review by awxiaoxian2020 — APPROVED** [review]
LGTM! Welcome aboard!

**Pull request merged**